lib/sen_tweet/bitfeels/metrics.ex

```diff
@@ -0,0 +1,49 @@
+defmodule SenTweet.Bitfeels.Metrics do
+  @moduledoc """
+  Handles Bitfeels Metric Events
+  """
+
+  alias SenTweet.Bitfeels.MetricServer
+  alias SenTweetWeb.MetricChannel
+
+  def handle_event([:bitfeels, :pipeline, :source], _measurements, metadata) do
```

Comment:
Each event will match on it's event name here, ex `[:bitfeels, :pipeline, :source]`. Then we have access to the measurements and metadata that we added in bitfeels.

Another metrics I'd like to add is average tweet processing time. So from the time that it hits the source pipeline to the time it leaves the sentiment pipeline. It bet it's close to the `senpy_response_time` :trollface:

Reply:
are you throwing shade, bruh?

Reply:
A little bit. To be honest, I'm impressed that we haven't made the flask server completely fall over. The response times aren't anywhere near great but maybe we can do something about that.